app/frontend/schools/settings/types/SchoolCustomize__Links.res

```diff
@@ -0,0 +1 @@
+type kind = HeaderLink | FooterLink | SocialLink
```

Comment:
Why is this required 🤔

Reply:
I was getting cyclic dependency errors. So I extracted type `kind`